indico/modules/events/editing/settings.py

```diff
@@ -18,7 +18,7 @@
     'editable_types': ['paper'],
 })
 
-_defaults = {'self_assign_allowed': False}
+_defaults = {'self_assign_allowed': False, 'submission_enabled': False, 'editing_enabled': False}
```

Comment:
I think this would look a bit nicer:

```python
_defaults = {
    'self_assign_allowed': False,
    'submission_enabled': False,
    'editing_enabled': False,
}
```